src/python/turicreate/test/test_image_similarity.py

```diff
@@ -12,9 +12,13 @@
 from turicreate.toolkits._internal_utils import _mac_ver
 import tempfile
 from . import util as test_util
+
+from turicreate.toolkits._main import ToolkitError as _ToolkitError
+from turicreate.toolkits.image_analysis.image_analysis import MODEL_TO_FEATURE_SIZE_MAPPING, get_deep_features
+
 import coremltools
 import numpy as np
-from turicreate.toolkits._main import ToolkitError as _ToolkitError
+from array import array
```

Comment:
I don't think this is being used.

Reply:
My bad, will remove it and push the corrected on in my next PR